docs/02-guides/configuration.mdx

```diff
@@ -1,14 +1,394 @@
 ---
 title: Inject configurations
 description:
-  This guide explains how to inject some configurations in Front-Commerce
+  Configuration is a part of software development that can quickly become messy.
+  Validation, detection, performance… there is a lot to take care of. In this
+  guide, we will see how Front-Commerce configuration works and how to create
+  new configurations or override existing ones.
 sidebar_position: 5
 ---
 
 <p>{frontMatter.description}</p>
 
-:::info WIP
+import ContactLink from "@site/src/components/ContactLink";
 
-see https://developers.front-commerce.com/docs/2.x/advanced/server/configurations
+In this guide we will see how they work and how to create new configurations or
+override existing ones.
```

Comment:
**issue:** it's similar to the last sentence of the description

**suggestion:** remove

```suggestion
```